packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts

```diff
@@ -0,0 +1,600 @@
+  expect(vaultAfter.batches).toStrictEqual([
+    { index: 0, collaterals: ['5000.00000000@DFI', '0.50000000@BTC'], loan: '500.00399539@TSLA' }, // refer to ln: 171, the last interest generated loanAmt divided by 2
+    { index: 1, collaterals: ['5000.00000000@DFI', '0.50000000@BTC'], loan: '500.00399539@TSLA' }
+  ])
+
+  const auctionsAfter = await bob.container.call('listauctions')
+  expect(auctionsAfter.length > 0).toStrictEqual(true)
+  expect(auctionsAfter[0].vaultId).toStrictEqual(bobVaultId)
+  expect(auctionsAfter[0].batchCount).toStrictEqual(2)
+  expect(auctionsAfter[0].liquidationHeight).toStrictEqual(168)
+  expect(auctionsAfter[0].liquidationPenalty).toStrictEqual(5)
+  expect(auctionsAfter[0].batches[0].collaterals).toStrictEqual(['5000.00000000@DFI', '0.50000000@BTC'])
+  expect(auctionsAfter[0].batches[0].loan).toStrictEqual('500.00399539@TSLA')
+
+  await tGroup.waitForSync()
+}
+
+describe('placeAuctionBid success', () => {
+  beforeEach(async () => {
+    await tGroup.start()
+    await alice.container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterEach(async () => {
+    await tGroup.stop()
+  })
+
+  it('should placeAuctionBid', async () => {
+    const bobColAccBefore = await bob.rpc.account.getAccount(bobColAddr)
+    expect(bobColAccBefore).toStrictEqual(['8900.00000000@DFI', '545.45454546@TSLA'])
+
+    const bobTSLAAccBefore = bobColAccBefore.length > 0
+      ? bobColAccBefore.find((amt: string) => amt.split('@')[1] === 'TSLA')
+      : undefined
+    const bobTSLAAmtBefore = bobTSLAAccBefore !== undefined ? Number(bobTSLAAccBefore.split('@')[0]) : 0
+
+    const txid = await bob.rpc.loan.placeAuctionBid({
+      vaultId: bobVaultId,
+      index: 0,
+      from: bobColAddr,
+      amount: '526@TSLA' // (500.00399539 * 5%) + 500 = 525.00419516, min first bid is 525.00419516
+    })
+    expect(typeof txid).toStrictEqual('string')
+    await bob.container.generate(1)
+    await tGroup.waitForSync()
+
+    const bobColAccAfter = await bob.rpc.account.getAccount(bobColAddr)
+    expect(bobColAccAfter).toStrictEqual(['8900.00000000@DFI', '19.45454546@TSLA'])
+
+    const bobTSLAAccAfter = bobColAccAfter.length > 0
+      ? bobColAccAfter.find((amt: string) => amt.split('@')[1] === 'TSLA')
+      : undefined
+    const bobTSLAAmtAfter = bobTSLAAccAfter !== undefined ? Number(bobTSLAAccAfter.split('@')[0]) : 0
+
+    expect(bobTSLAAmtBefore - bobTSLAAmtAfter).toStrictEqual(526) // 545.45454546 - 19.45454546 = 526
+
+    const aliceColAccBefore = await alice.rpc.account.getAccount(aliceColAddr)
+    expect(aliceColAccBefore).toStrictEqual(['30000.00000000@DFI', '29999.00000000@BTC', '10000.00000000@TSLA'])
+
+    // test second round placeAuctionBid
+    await alice.rpc.loan.placeAuctionBid({
+      vaultId: bobVaultId,
+      index: 0,
+      from: aliceColAddr,
+      amount: '535@TSLA' // (526 * 1%) + 525 = 531.26
+    })
+    await alice.generate(1)
+    await tGroup.waitForSync()
+
```

Comment:
Can check Bob's TSLA is released after Alice place higher bid.

Reply:
we have that checker ady
https://github.com/DeFiCh/jellyfish/blob/77928df72a1684642cc1a61341d136261bd1364b/packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts#L297-L298